terragrunt/aws/s3/s3.tf

```
@@ -0,0 +1,38 @@
+module "s3_buckets" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v5.1.8//S3"
+  for_each          = var.buckets
+  bucket_name       = each.key
+  billing_tag_value = var.billing_code
+}
```

You'll also need a `logging` config block to send S3 access logs to the log bucket:
```suggestion
module "s3_buckets" {
  source      = "github.com/cds-snc/terraform-modules?ref=v5.1.8//S3"
  for_each    = var.buckets
  bucket_name = each.key
  
  logging = {
    target_bucket = module.log_bucket.s3_bucket_id
  }
  
  billing_tag_value = var.billing_code
}
```